src/main/java/ser/mil/rideapp/controller/DriverController.java

```diff
@@ -0,0 +1,36 @@
+package ser.mil.rideapp.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ser.mil.rideapp.controller.request.DriverRequest;
+import ser.mil.rideapp.domain.model.Provider;
+import ser.mil.rideapp.domain.service.DriverService;
+
+@RestController
+@RequestMapping("/driver")
+public class DriverController {
+    private final DriverService driverService;
+
+    @Autowired
+    public DriverController(DriverService driverService) {
+        this.driverService = driverService;
+    }
+    @PostMapping("/create")
+    public void addDriver(DriverRequest driverRequest) {
+        driverService.addDriver(
+                driverRequest.firstName(),
+                driverRequest.lastName(),
+                driverRequest.available(),
```

Comment:
Czy miałaby się kryć jakaś większa logika za przekazywanie "available" przez request? Obecnie jak ktoś stworzy driver z available == false to nigdy nie stanie się dostępny. Myślę że lepiej pozbyć się available z requesta i domyslnie ustawic go jako true